common/messages.py

```diff
@@ -129,12 +129,13 @@ def __init__(self, version: str):
         self.version = version
 
 
-# Example json: { 'unique_id' : 123, 'ip' : '1.2.3.4', 'eligible_for_first_win': false }
+# Example json: { 'unique_id' : 123, 'display_name': 'playername' 'ip' : '1.2.3.4', 'eligible_for_first_win': false }
 class Login2LauncherAddPlayer(Message):
```

Comment:
Don't forget to increase the major version number of the Login2Launcher protocol when changing the protocol in an incompatible way like this.